Monika After Story/game/script-moods.rpy

```diff
@@ -556,3 +556,72 @@ label mas_mood_yearolder_years:
 label mas_mood_yearolder_leap_today:
     # nothing special occurs here for now
     jump mas_mood_yearolder_bday_true
+
+init 5 python:
+    addEvent(Event(persistent._mas_mood_database,"mas_mood_scared",prompt="scared",category=[store.mas_moods.TYPE_BAD],unlocked=True),eventdb=store.mas_moods.mood_db)
+
+label mas_mood_scared:
+    m 1r "{i}There is nothing to fear, but fear itself.{/i}"
+    m 1k "Ahaha~ I'm kidding."
+    m 3a "Though the reason we become scared, is because we sense any form of danger."
+    m "This can be in a physical or psychological form."
+    m 1d "Fear is a survival instinct that we are born with, and this trait keeps us from harm."
+    m 1g "Though, this instinct can also restrict our actions that may be benefitial to us, instead of damaging."
+    m 1e "I hope your reasons behind being scared, is keeping you from harm [player]."
+    m 1f "I would never want to see my beloved sweetheart hurt."
+    return
+
+init 5 python:
+    addEvent(Event(persistent._mas_mood_database,"mas_mood_guilty",prompt="guilty",category=[store.mas_moods.TYPE_BAD],unlocked=True),eventdb=store.mas_moods.mood_db)
+
+label mas_mood_guilty:
+
+    m 1f "Aw! You shouldn't feel guilty [player]."
+    m 2g "There is no way that you could know or do anything so bad to feel that way."
+    m 4p "I mean, doing what I did to my friends was something to feel guilty over."
```

Comment:
Should probably have her reference herself in this line, otherwise it sounds like the player did it and should feel guilty over it.
Such as...
"I mean, doing what I did to my friends was something I sometimes feel guilty over." 
It still works by itself, but can be easily misinterpreted.